internal/provider/atlas_migration_resource_test.go

```diff
@@ -549,21 +549,24 @@ func TestAccMigrationResource_AtlasURL(t *testing.T) {
 			}
 		}))
 		config = fmt.Sprintf(`
-		provider "atlas" {
+		data "atlas_migration" "hello" {
+			url = "%[2]s"
+			dir = "atlas://test"
 			cloud {
 				token   = "aci_bearer_token"
 				url     = "%[1]s"
 				project = "test"
 			}
 		}
-		data "atlas_migration" "hello" {
-			url = "%[2]s"
-			dir = "atlas://test"
-		}
 		resource "atlas_migration" "testdb" {
 			url     = "%[2]s"
 			version = data.atlas_migration.hello.next
 			dir     = data.atlas_migration.hello.dir
+			cloud {
```

Comment:
Moving `cloud` into `resource` | `data` to bypass the `login` check. Login will be covered by integration tests